app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    mv, file_1, path_file_2 = split_command
+
+    if "/" in command:
```

Comment:
Code Efficiency: You check if the command contains a slash ('/'), but this does not guarantee that the command is a valid 'mv' command. It's better to validate the command structure more thoroughly.